src/lib/server/db/schema.ts

```diff
@@ -23,6 +47,36 @@ export * from './users.sql';
 export * from './votes.comments.sql';
 export * from './votes.posts.sql';
 
+/**
+ * This is a list of all of our database tables
+ * It is essential that this list be kept up to date to guarantee test isolation
+ */
+export const tables = [
+    channelBannedUserTable,
+    userBlockTable,
+    subscriptionTable,
+    commentTable,
+    followTable,
+    inviteTable,
+    playlistPostTable,
+    playlistTable,
+    publicChannelTable,
+    channelPostReportTable,
+    channelReportTable,
+    channelUserReportTable,
+    userReportTable,
+    roleTable,
+    userRoleTable,
+    sessionTable,
+    channelTagsTable,
+    postTagTable,
+    commentVoteTable,
+    postVoteTable,
+    postTable,
+    channelTable,
+    userTable,
+];
```

Comment:
It'll likely be kinda easy for people to forget to add new tables to this list. It'd be sick to define a custom lint or a test to just validate that the length of this list is equal to the number of tables that we have.

Reply:
I think that would be nice, but definitely not important enough to block this PR.

Reply:
I was able to avoid this entirely

Reply:
Very snazzy